README.md

```diff
@@ -27,9 +27,88 @@ While setting up cassandra instance, make sure the data directories of cassandra
 Testing
 -------
 
-We rely on docker containers for integration tests.
+We rely on the in-jvm dtest framework for testing. You must manually build the dtest jars before you start integration tests.
+At the moment, the JMX feature is unreleased in Cassandra, so you can use the following to build from the PR branches:
 
-The only requirement is to install and run [Docker](https://www.docker.com/products/docker-desktop/) on your test machine.
+```shell
+./scripts/build-dtest-jars.sh
+```
+
+The build script supports two parameters:
+- `REPO` - the Cassandra git repository to use for the source files. This is helpful if you need to test with a fork of the Cassandra codebase.
+  - default: `git@github.com:apache/cassandra.git`
+- `BRANCHES` - a space-delimited list of branches to build.
+  -default: `"cassandra-3.11 cassandra-4.1"`
```

Comment:
Just a nit: I think the actual defaults on the script are different?

Reply:
Good point - we were originally going to default to 3.11 and 4.1 for testing, but ended up changing our minds, so I'll fix this.